jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Util.java

```diff
@@ -139,4 +140,16 @@ static void getTargetInformation(AttachProvider theProvider, VirtualMachineDescr
 		}
 	}
 
+	/**
+	 * Print the Properties object in key-value format.
+	 * This differs from Properties.list in that is does not truncate long values.
+	 * @param out
+	 * @param props
+	 */
+	static void printProperties(PrintStream out, Properties props) {
+		for (Object key: props.keySet()) {
```

Comment:
I expect it's more efficient to iterate over the Map.Entry's of the props.entrySet().